python/cudf/cudf/tests/test_stats.py

```diff
@@ -387,3 +387,19 @@ def test_df_corr():
     got = gdf.corr()
     expected = pdf.corr()
     utils.assert_eq(got, expected)
+
+
+def test_nans_stats():
+    from cudf.tests import utils
+
+    data = [0.0, 1, 3, 6, np.NaN, 7, 5.0, 5, 2, 3, -100]
+    psr = pd.Series(data)
+    gsr = Series(data, nan_as_null=False)
+
+    utils.assert_eq(psr.mean(), gsr.mean())
+    utils.assert_eq(psr.min(), gsr.min())
+    utils.assert_eq(psr.max(), gsr.max())
+    utils.assert_eq(psr.sum(), gsr.sum())
+    utils.assert_eq(psr.product(), gsr.product())
+    utils.assert_eq(psr.var(), gsr.var())
+    utils.assert_eq(psr.std(), gsr.std())
```

Comment:
I would pytest parameterize over these different ops, it makes a little easier to localize issues when debugging.

Reply:
@brandon-b-miller, Done 👍